CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## master / unreleased
 
+## 0.5.0-rc1 / 2020-01-15
+
+Note that the ruler flags need to be changed in this upgrade. You're moving from a single node ruler to something that might need to be sharded.
+If you are running with a high `-ruler.num-workers` and if you're not able to execute all your rules in 15s, then you'll need to shard.
```

Comment:
Where does this `15s` come from?

Reply:
It's the default for `-ruler.evaluation-interval`; I hadn't noticed it was configurable.

Reply:
Then it could be more clear if we mention it here.